cmd/utils/flags.go

```diff
@@ -123,3 +123,15 @@ var (
 		Usage: "Specify account type as secp256k1",
 	}
 )
+
+// FixFlagsOder allow us to use various flag order formats, eg: (gossamer init --config config.toml and  gossamer --config config.toml init)
+func FixFlagsOder(f func(ctx *cli.Context) error) func(*cli.Context) error {
```

Comment:
should be `FixFlagOrder`?

Reply:
Sure, I mean it fixes many flags, so that why the plural there, but can change.